readme.md

```diff
@@ -848,6 +866,7 @@ Yields:
 {
   "type": "definition",
   "identifier": "alpha",
+  "definition": "Alpha",
```

Comment:
Reference? Why definition? 🤔

Reply:
Because it is not a reference?  `identifers` are present in xxxReference and xxxDefinition nodes.  For the former, I added `reference` properties, for the latter, `definition`.  Make sense?

Reply:
Oh sorry, I missed your note. I'm fine with either but have a slight preference for reference, as it's more in line with identifier (referencing something unique) though 🤔

Reply:
@rubys Are you willing to use `reference`, or do you strongly prefer `definition`?